### ISSUE TITLE
Add fields to all entity modes

### ISSUE THREAD

**Issue by anotherjames** [computerminds/cm_config_tools#20]
for example, when a field is added in the UI, it's added to the default mode, great. and it will be hidden in all the others by default, but it won't actually make it into the config for export until those other modes have actually been saved, which is a bit annoying.

**Comment by darthsteven** [comment]
Is this more of a core issue than a cm_config_tools one?

**Comment by anotherjames** [comment]
Quite probably, yes. But core issues move slowly, cm_config_tools allows us to get fixes in that can then be removed when core has them fixed.

Having said that, cm_config_tools now moves fairly slowly too as I'm tied up with client work :-)

**Comment by darthsteven** [comment]
> which is a bit annoying

Is it just annoying or is it totally broken OMG we should fix this before something really bad happens?

If the former, then we could just try and fix in core and that would be good enough I reckons.

**Comment by anotherjames** [comment]
Yes, I agree fixing in core is the best. cm_config_tools can just hold an equivalent fix too whilst we wait for any fix in core to get included in a stable release.
